docs/design.md

```diff
@@ -63,29 +62,32 @@ The meta.json is updated during upload time on sidecars.
               │                   Object Storage                 │
               └──────────────────────────────────────────────────┘
 ```
+
 #### Chunk
-A chunk is part of the data structure of Prometheus TSDB, holding up to 120 samples for a single timeseries.
+
+A chunk is part of the data structure of Prometheus TSDB, holding up to 120 samples for a single time series.
 
 #### Chunk file
-A chunk file is the file in TSDB block that contains up to 0.5 GB worth of chunk entries in binary format.
+
+A chunk file is a file in a TSDB block that contains up to 0.5 GB worth of chunk entries in binary format.
 
 ### Stores
 
 A store node acts as a gateway to block data that is stored in an object storage bucket. It implements the same gRPC API as data sources to provide access to all metric data found in the bucket.
 
 It continuously synchronizes which blocks exist in the bucket and translates requests for metric data into object storage requests. It implements various strategies to minimize the number of requests to the object storage such as filtering relevant blocks by their metadata (e.g. time range and labels) and caching frequent index lookups.
 
-The Prometheus 2.0 storage layout is optimized for minimal read amplification. For example, sample data for the same time series is sequentially aligned in a [chunk file](design.md/#chunk-file). Similarly, series for the same metric name are sequentially aligned as well.
+The Prometheus 2.0 storage layout is optimized for minimal read amplification. For example, sample data for the same time series are sequentially aligned in a [chunk file](design.md/#chunk-file). Similarly, series for the same metric name is sequentially aligned as well.
```

Comment:
Sample data `is` not `are` I think? (: The same in second sentence `series` is used here as single series.